index.cjs

```diff
@@ -15,6 +15,26 @@ function Argv(processArgs, cwd) {
   return argv;
 }
 
+function defineGetter(obj, key, getter) {
+  Object.defineProperty(obj, key, {
+    configurable: true,
+    enumerable: true,
+    get: getter,
+  });
+}
+function lookupGetter(obj, key) {
+  while (obj) {
+    const desc = Object.getOwnPropertyDescriptor(obj, key);
+    if (typeof desc !== 'undefined') {
+      if (desc.get) {
+        return desc.get;
```

Comment:
@bmeck can we replace this with:

```js
function lookupGetter(obj, key) {
  const desc = Object.getOwnPropertyDescriptor(obj, key);
  if (typeof desc !== 'undefined') {
    return desc.get;
  }
}
```

So that coverage remains at 100%? I'm assuming that `desc.get` will either be the getter function, or `undefined`?

Reply:
I did a pass to see if it was possible to go through a proto and it only occurs on extremely exotic and tailored Proxy stuff so this replacement should be fine due to it being invoked against the keys/own prop keys.